### DISCUSSION TITLE
Proxmox VE TurnKey Nextcloud VM

### DISCUSSION THREAD

**Discussion by roswitina** [tteck/Proxmox#2351] [Q&A] [answered]
I am unable to install a "Proxmox VE TurnKey Nextcloud VM" with this script. As far as I have been able to determine, only 1 drive with 798M is created.

Is this correct and if so, what happens next?

I have attached two screenshots of the installation.

![grafik](https://github.com/tteck/Proxmox/assets/2253471/07fa6edb-f960-48cf-b468-6c12099aa8d3)
![grafik](https://github.com/tteck/Proxmox/assets/2253471/52694842-1eec-4a24-b8de-f3fae8dfc962)

**Comment by roswitina** [comment]
The VM has the IP 192.168.1.125

![grafik](https://github.com/tteck/Proxmox/assets/2253471/9d298f3c-5dd1-4b3e-beb5-1662c7d45b8b)

After boot the vm and use Installing from proxmox console...

Guided 
![grafik](https://github.com/tteck/Proxmox/assets/2253471/f06b475d-7fb2-4e02-b9d5-92fde276486f)
![grafik](https://github.com/tteck/Proxmox/assets/2253471/b8db7c7b-caaa-4fa7-8044-4193ce69b3fa)

![grafik](https://github.com/tteck/Proxmox/assets/2253471/af0fcced-3eb9-4066-9def-335a9ce020a2)

Manual
![grafik](https://github.com/tteck/Proxmox/assets/2253471/1b036800-41d8-482d-8940-44ba62d12f07)

The harddisk is 836,8 MB
What should Turnkey be installed on?
No disk has been created.

**Reply by tteck** [reply, marked as answer]
Removed from the website until more testing can be done.
